events/beam-robotics.html

```diff
@@ -0,0 +1,166 @@
+</p>
+
+<hr class="divider" />
+
+<div class="secondary-top-section">
+    <p class="title text-2x no-margin">
+        <b>Setup</b>
+    </p>
+    <p>
+        In these lessons, we will learn how to code <a class="tla-link" href="https://www.makeblock.com/mbot" target="_blank" rel= "noopener noreferrer" ><b>mBots</b></a>! 
+        Before these lessons, you may want to <b>acquire an <a class="tla-link" href="https://www.makeblock.com/mbot" target="_blank" rel= "noopener noreferrer" >mBot</b></a>
+         which you will learn to program, and <a class="tla-link" href="https://mblock.makeblock.com/en-us/download/" target="_blank" rel= "noopener noreferrer" ><b>download mBlock</b></a> which will be where you code your robot.
+    </p>
+    <div class ="bottom-button-container">
+        <a class="button button-dark" href="https://www.makeblock.com/mbot" target="_blank" rel="noopener noreferrer"> <span class="fa fa-link"></span> mBot purchase Link</a>
+        <a class="button button-dark" href="https://mblock.makeblock.com/en-us/download/" target="_blank" rel="noopener noreferrer"> <span class="fa fa-link"></span> mBlock Download Guide</a>
+    </div>
+</div>
+
+<hr class="divider" />
+
+<div class="secondary-top-section">
+    <p class="title text-2x no-margin">
+        <b>Lessons</b>
+    </p>
+</div>
+
+<div class="row-desktop row-desktop-ratio-1-2 my-3">
+    <div class="grid-col">
+      <img class="img-responsive" src="{{site.baseurl}}/img/beam-robotics/monkey.svg" alt="a monkey from a game you'll code in our week 1 lesson, sitting under a bananna tree"/>
+    </div>
+    <div>
+      <h2 class="title text-1.5x">
+        <a class="tla-link" href="#" target="_blank" rel="noopener noreferrer">Week 1: Intro to Coding</a>  
+      </h2>
+      <p>
+        How do people make games and websites and all that cool stuff? Learn the basics of <b>coding</b> with mBlock and block code! By the end of this lesson, you will learn how to make <b> <a class="tla-link" href="https://planet.mblock.cc/project/423512" target="_blank" rel= "noopener noreferrer" >this spicy game</b> </a>. 
+    </p>
+    <!---
+      <div class="button-container">
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fab fa-youtube"></span> Video</a>
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fas fa-book"></span> Student Guide</a>
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fas fa-chalkboard-teacher"></span> Instructor Guide</a>
+      </div>
+      -->
+    </div>
+  </div>
+
```

Comment:
add divides throughout so itll probably end up looking something like [this](https://drive.google.com/file/d/1hEyD_cV3VCwZWgNaGa742NB7S6Vyz7_l/view)
```suggestion
<hr class="divider" />
```